src/app/backend/replicasetdetail.go

```diff
@@ -158,7 +162,7 @@ func DeleteReplicaSetWithPods(client *client.Client, namespace string, name stri
 }
 
 // Returns detailed information about service from given service
-func getServiceDetail(service api.Service) ServiceDetail {
+func GetServiceDetail(service api.Service) ServiceDetail {
```

Comment:
I didn't knew about it to be honest, but now I will remember, thanks. Function should be private I think, I will change it back.